app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -457,6 +431,9 @@ class ThreadAdapter : RecyclerView.Adapter<ThreadViewHolder>(), RealmChangesBind
 
         private var doesWebViewNeedInit = true
 
+        var alreadySplitBody: String? = null
+        var alreadySplitQuote: String? = null
```

Comment:
It looks like a boolean, you should choose other names, such as `splittedBody` and `splittedQuote`.